src/modules/field/field.js

```diff
@@ -583,6 +583,9 @@ function options_field_widget_form(form, form_state, field, instance, langcode,
                 if (items[delta].required) {
                   delete items[delta].options[''];
                 }
+                if (items[delta].item && typeof items[delta].item.value !== 'undefined') {
+                  items[delta].value = items[delta].value = items[delta].item.value;
```

Comment:
@jacobembree I'm having a hard time understanding this line here, would you please explain the two `=` signs, I don't believe I've seen that before in JavaScript? Thanks.